vm/src/stdlib/winapi.rs

```diff
@@ -235,29 +289,29 @@ mod _winapi {
 
         let procinfo = unsafe {
             let mut procinfo = std::mem::MaybeUninit::uninit();
-            let ret = processthreadsapi::CreateProcessW(
-                app_name,
-                command_line,
+            let ret = Threading::CreateProcessW(
+                windows::core::PCWSTR(app_name),
+                windows::core::PWSTR(command_line),
                 null_mut(),
                 null_mut(),
-                args.inherit_handles,
+                BOOL(args.inherit_handles),
                 args.creation_flags
-                    | winbase::EXTENDED_STARTUPINFO_PRESENT
-                    | winbase::CREATE_UNICODE_ENVIRONMENT,
+                    | Threading::EXTENDED_STARTUPINFO_PRESENT
+                    | Threading::CREATE_UNICODE_ENVIRONMENT,
```

Comment:
As we cant or operation between `u32` and `struct(u32)`, we need to do like this
```rust
PROCESS_CREATION_FLAGS(args.creation_flags
                    | Threading::EXTENDED_STARTUPINFO_PRESENT.0
                    | Threading::CREATE_UNICODE_ENVIRONMENT.0)
```